ocaml/xapi/xapi_pool.ml

```diff
@@ -3596,3 +3596,12 @@ let set_uefi_certificates ~__context ~self ~value =
         )
         (Db.Host.get_all ~__context)
   )
+
+let set_https_only ~__context ~self ~value =
+  Helpers.call_api_functions ~__context (fun rpc session_id ->
+      List.iter
+        (fun host ->
+          Client.Host.set_https_only ~rpc ~session_id ~self:host ~value
+        )
+        (Db.Host.get_all ~__context)
```

Comment:
Note: it doesn't look like order matters in this case

Reply:
The calls from this host (the coordinator) to all other hosts are always done over HTTPS, so there is no risk of the side-effects of this function caution harm to the completion of the call itself.